Realm/ObjectServerTests/RLMObjectServerTests.mm

```diff
@@ -17,17 +17,13 @@
 ////////////////////////////////////////////////////////////////////////////
 
 #import "RLMSyncTestCase.h"
+#import "RLMTestUtils.h"
 #import "RLMSyncSessionRefreshHandle+ObjectServerTests.h"
 #import "RLMSyncUser+ObjectServerTests.h"
 #import "RLMSyncUtil_Private.h"
 #import "RLMRealmConfiguration_Private.h"
 #import "RLMRealmUtil.hpp"
 
-#define ACCOUNT_NAME() NSStringFromSelector(_cmd)
```

Comment:
Why remove this helper? It hurts code readability and makes this patch larger for no apparent reason.